client/packages/coldchain/src/Equipment/DetailView/Tabs/Summary.tsx

```diff
@@ -280,6 +293,16 @@ export const Summary = ({ draft, onChange, locations }: SummaryProps) => {
             />
           </Row>
         </Section>
+        <Section heading={t('label.donor')}>
+          <Row label={t('label.donor')}>
+            <DonorSearchInput
+              value={draft.donor as NameRowFragment} // Using as NameRowFragment is ok, because the comparison function is based on the id
+              onChange={e => onChange({ donor: e, donorNameId: e?.id })}
```

Comment:
BTW: I think we could probably get away with just the onChange function if we implemented the full Mui Types.

```
Callback fired when the value changes.

Signature:
function(event: React.SyntheticEvent, value: Value | Array, reason: string, details?: string) => void
event The event source of the callback.
value The new value of the component.
reason One of "createOption", "selectOption", "removeOption", "blur" or "clear".
```
https://mui.com/material-ui/api/autocomplete/